code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -29,6 +29,7 @@
 	//var/list/viruses = list()
 	var/color = "#000000" //rgb: 0, 0, 0 (does not support alpha channels - yet!)
 	var/alpha = 255
+	var/dupeable = TRUE	//whether the reagent can be duplicated by standard reagent duplication methods such as a service borg shaker or odysseus
```

Comment:
Nearly all the values stored in reagents are multipliers, they cannot be bitflags

Except for like, this one and 'overdose' which is redundant anyways